crates/holochain/src/core/ribosome/host_fn/debug.rs

```diff
@@ -9,17 +9,16 @@ use tracing::*;
 pub fn debug(
     _ribosome: Arc<impl RibosomeT>,
     _call_context: Arc<CallContext>,
-    input: DebugInput,
-) -> RibosomeResult<DebugOutput> {
-    let msg: DebugMsg = input.into_inner();
+    input: DebugMsg,
+) -> RibosomeResult<()> {
```

Comment:
see above

Reply:
this will never have an output because it is forwarding to debugging machinery on the host and debugs don't have return values